site/assets/scss/_component-examples.scss

```diff
@@ -217,6 +217,38 @@
   }
 }
 
+.bd-example-stack-utils {
+  position: relative;
+  height: 162px;
+
+  [class*="z-index"] {
+    position: absolute;
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+
+  .z-index-n1 {
+    top: 24px;
+    left: 24px;
+  }
+
+  .z-index-0 {
+    top: 48px;
+    left: 48px;
+  }
+
+  .z-index-1 {
+    top: 72px;
+    left: 72px;
+  }
+
+  .z-index-2 {
+    top: 96px;
+    left: 96px;
```

Comment:
Use `rem` for those examples, it'd be more responsive if a user changes his base font size in his browser settings.

Reply:
done!

Reply:
done!